db/migrate/20150421191213_add_column_to_company.rb

```diff
@@ -0,0 +1,6 @@
+class AddColumnToCompany < ActiveRecord::Migration
+  def change
+    add_column :companies, :hiring, :boolean
```

Comment:
We briefly discussed having hiring being a status that admin could control.  Submitted, Emailed, Called, Hiring?  Just a thought.